app/models/config.rb

```diff
@@ -24,6 +24,7 @@ class Config < ApplicationRecord
     time_zone: "Time zone to use for displaying dates. Default is Eastern. Valid options are Eastern, Central, Mountain, Pacific, Alaska, Hawaii, Arizona, Indiana (East), or Puerto Rico.",
     procedure_type: "Any kind of distinction in procedure your fund would like to track. Field hides if no options " \
                     "are added here. Please separate with commas.",
+    display_practical_support_attachment_url: 'CAUTION: Whether or not to allow people to enter attachment URLs for practical support entries; for example, a link to a file in Google Drive. Please ensure that any system storing these is properly secured by your fund!',
```

Comment:
i def still feel wary about this, but I think this is the right balance of empowering vs security

Reply:
I also feel ways about this but yeah I think this is an okay balance